### PR TITLE
Add rune images to abyss overlay

### PR DESCRIPTION
**Pull request by tylerthardy**
image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/runecraft/AbyssOverlay.java [hunk 1]
```diff
@@ -128,35 +125,61 @@ private void renderRifts(Graphics2D graphics, DecorativeObject object)
...
-	public BufferedImage getImage(AbyssRifts rift)
+	public BufferedImage getOverlayImage(AbyssRifts rift)
+	{
+		BufferedImage image = abyssOverlayIcons.get(rift);
+		if (image != null)
+		{
+			return image;
+		}
+
+		image = itemManager.getImage(rift.getItemId());
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/runecraft/AbyssOverlay.java]: The item manager already has its own cache so there is no reason to do this here.

**Comment by deathbeam** [comment]
Any update to this?

**Comment by deathbeam** [comment]
Needs rebase, and resolving the review comments, otherwise this should be closed.

**Comment by deathbeam** [comment]
Closing as this is very old PR now and has a lot of merge conflicts, reopen if you want to re-implement it.

**Pull request closed without merging**